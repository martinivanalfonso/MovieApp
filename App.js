import { NavigationContainer } from '@react-navigation/native'
import React from 'react'
import {
  StatusBar,
  useColorScheme
} from 'react-native'

import Navigation from './app/Navigation'

const App = () => {
  const isDarkMode = useColorScheme() === 'dark'

  return (
    <NavigationContainer>
      <StatusBar barStyle={isDarkMode ? 'light-content' : 'dark-content'} />
      <Navigation />
    </NavigationContainer>
  )
}

export default App
