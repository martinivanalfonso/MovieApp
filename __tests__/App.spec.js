import React from 'react'
import { shallow } from 'enzyme'
import App from '../App'

describe('App Component', () => {
  test('renders correctly', () => {
    const component = shallow(<App />)
    expect(component).toMatchSnapshot()
  })
})
