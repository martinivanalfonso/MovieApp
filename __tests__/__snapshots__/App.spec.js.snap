// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App Component renders correctly 1`] = `
<ForwardRef(NavigationContainer)>
  <StatusBar
    animated={false}
    barStyle="dark-content"
    showHideTransition="fade"
  />
  <Navigation />
</ForwardRef(NavigationContainer)>
`;
